warehouse/core/src/main/java/datawave/util/TableName.java

```diff
@@ -0,0 +1,23 @@
+package datawave.util;
+
+/**
+ * Class that contains default table names
+ */
+public final class TableName {
+    
```

Comment:
you need a private CTOR:

private TableName() {}

Reply:
Added in next commit